### PR TITLE
File upload

### PR DESCRIPTION
**Pull request by juanfabrega**
File upload component.

Screen Shot 2020-10-20 at 11 28 14 AM

### CODE REVIEW

**Review by nathanyoung — CHANGES_REQUESTED** [review]
Looks good, but should we add it to the Formik Form example?

What about `isRequired`

**Comment by juanfabrega** [comment]
> Looks good, but should we add it to the Formik Form example?
> 
> What about `isRequired`

Yup, definitely a good add. Will update and re-request review in a bit.

**Comment by nathanyoung** [comment]
This story is rendering a `button` when it supposed to render a `<a>`

**Comment by nathanyoung** [comment]
@juanfabrega The `isRequired` story results in this when pressing the submit button.

Screen Shot 2020-10-21 at 2 01 06 PM

File: src/components/Button/Button.test.jsx [hunk 1]
```diff
@@ -142,9 +143,9 @@ describe('Button', () => {
         expect(buttonElement).toBeInTheDocument();
       });
 
-      test('it renders an anchor tag if an href attribute is passed as a prop', () => {
+      test('it renders an anchor tag if as prop `a` is passed', () => {
```
Comment (nathanyoung) [inline src/components/Button/Button.test.jsx:146]: this test is passing, but Chromatic says differently: https://www.chromatic.com/snapshot?appId=5ed9214b642dc10022b50a2d&id=5f9080a3409e4d0022907fcb

**Review by nathanyoung — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.test.jsx (hunk 1 above)
Comment (juanfabrega) [reply src/components/Button/Button.test.jsx:146]: @nathanyoung I neglected to update the chromatic story, but the test has the correct implementation. Story is updating now.

**Review by nathanyoung — APPROVED** [review]

**Pull request merged**